### PR TITLE
Added ArchangelFryingPan :P

### PR DESCRIPTION
**Pull request by VaibhavBajaj**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,6 +51,7 @@ Credits for images:
 * Steel Armor: https://pixabay.com/en/shield-swords-knight-medieval-309126/(Resized)
 * Lucky Copper Sword: https://openclipart.org/detail/2664/sword-or-yellow
 * LegendaryBrahmastra: https://pixabay.com/en/bow-arrow-shaft-bolt-weapon-sport-307275/
+* ArchmageFryingPan: https://pixabay.com/en/pan-cook-sear-frying-pan-554072/
```
Comment (macalinao) [inline README.md]: Archangel*

**Comment by macalinao** [comment]
You should remove `.DS_Store`

File: README.md (hunk 1 above)
Comment (VaibhavBajaj) [reply README.md]: Resolved

**Comment by VaibhavBajaj** [comment]
Resolved

**Comment by Avalancs** [comment]
Well Done!

Merged in 8b491bb0600a31b13cde94a4e49cccc836905046

**Pull request closed without merging**